app/ethtest_helper.go

```diff
@@ -225,7 +225,7 @@ func genesisStateWithValSet(codec codec.Codec, genesisState simapp.GenesisState,
 	associations := []delegationtypes.StakerToOperator{
 		{
 			Operator: operator.String(),
-			StakerID: stakerID,
+			StakerId: stakerID,
```

Comment:
To avoid the Lint errors from Golang CI, I changed the name of such fields to Staker. I don't know if they're still triggered.